src/vaadin-combo-box-light.html

```diff
@@ -74,16 +80,18 @@
      * </vaadin-combo-box-light>
      * ```
      * @memberof Vaadin
+     * @mixes Vaadin.ControlStateMixin
      * @mixes Vaadin.ComboBoxDataProviderMixin
      * @mixes Vaadin.ComboBoxMixin
      * @mixes Vaadin.ThemableMixin
      * @mixes Vaadin.ThemePropertyMixin
      */
     class ComboBoxLightElement extends
-      Vaadin.ThemePropertyMixin(
-        Vaadin.ThemableMixin(
-          Vaadin.ComboBoxDataProviderMixin(
-            Vaadin.ComboBoxMixin(Polymer.Element)))) {
+      Vaadin.ControlStateMixin(
```

Comment:
Not a good idea to have `Vaadin.ControlStateMixin` here, where the host element does not need to delegate focus into shadow DOM, the input is in the light DOM and can receive focus naturally.

This brings a lot of workarounds over things that should work natively already, and this is risky.

In particular, because the mixin makes the focus element focusable, when tabbing into `<vaadin-combo-box-light><input></vaadin-combo-box-light>`, there is an extra step with a focus frame around `<vaadin-combo-box-light>`.